### PR TITLE
Npetrovic/bw ops sweeps

### LINKED ISSUE #4775: Add sweep tests for bw operations

### PR DESCRIPTION
**Pull request by npetrovic-tenstorrent**
Added backward operation sweeps:
- log_bw
- rsub_bw
- abs_bw
- sqrt_bw
- neg_bw
- lt_bw
- gt_bw
- relu_bw
- ne_bw
- clamp_bw
- clamp_min_bw
- clamp_max_bw
- binary_le_bw

 and unit tests for the ones which are failing on GS (clamp, clamp min, clamp max, sqrt)

### CODE REVIEW

File: tests/tt_eager/python_api_testing/sweep_tests/test_configs/ci_sweep_tests_broken/wormhole/pytorch_backward_gt.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          low: -100
+          high: 100
+      comparison:
+        function: comp_pcc_list
+      args-gen: gen_dtype_layout_device
+      args:
+        data-layout: ["TILE"]
+        data-type: ["BFLOAT16"]
+        buffer-type: ["DRAM", "L1", "SYSTEM_MEMORY"]
+        out-buffer-type: ["DRAM"]
```
Comment (nemanjagrujic) [inline tests/tt_eager/python_api_testing/sweep_tests/test_configs/ci_sweep_tests_broken/wormhole/pytorch_backward_gt.yaml:24]: L1?

**Comment by nemanjagrujic** [comment]
@nemanjagrujic This pull request includes changes for div and div_unary which are covered by #4875 pull req.

**Comment by nemanjagrujic** [comment]
@npetrovic-tenstorrent Yaml for rsub for Grayskull card is not in working folder nor in broken folder.

File: tests/tt_eager/python_api_testing/sweep_tests/test_configs/ci_sweep_tests_broken/wormhole/pytorch_backward_gt.yaml (hunk 1 above)
Comment (npetrovic-tenstorrent) [reply tests/tt_eager/python_api_testing/sweep_tests/test_configs/ci_sweep_tests_broken/wormhole/pytorch_backward_gt.yaml]: Moreover, div-related changes were removed

**Review by nemanjagrujic — APPROVED** [review]

**Pull request merged**